pkgs/tools/audio/bpm-tools/default.nix

```diff
@@ -12,14 +22,22 @@ stdenv.mkDerivation rec {
     sha256 = "151vfbs8h3cibs7kbdps5pqrsxhpjv16y2iyfqbxzsclylgfivrp";
   };
 
+  nativeBuildInputs = [ makeWrapper ];
+
   patchPhase = ''
     patchShebangs bpm-tag
     patchShebangs bpm-graph
   '';
 
-  installFlags = [
-    "PREFIX=${placeholder "out"}"
-  ];
+  installPhase = ''
+    install -D bpm $out/bin/bpm
```

Comment:
Why not let it install like before and do the wrapping in `postFixup`? This way when a new program is added to the project, we will not forget to install it.